### ISSUE TITLE
bad sql for oracle db

### ISSUE THREAD

**Issue by MichaelReicheOracle** [waarp/WaarpR66#205]
02/12/2016/16:16:54.902 ERROR [org.waarp.common.database.DbPreparedStatement] [main] org.waarp.common.database.DbPreparedStatement.executeQuery(DbPreparedStatement.java:282) : SQL Exception executeQuery:SELECT count(*) FROM  HOSTS  WHERE ISPROXIFIED = true ORA-00904: "TRUE": invalid identifier

**Comment by fredericBregier** [comment]
Yes due to a non ascendant compatibility of Oracle... and non compatibility with other databases...

This issue among others with Oracle are known. Thank you for reporting!

**Comment by bcarlin** [comment]
There are a lot of issues with Oracle for a small part of the user base... We will fix it, but it is not a priority for us.
However, we accept PR if you have a fix for that.
